src/networkDetails.ts

```diff
@@ -20,8 +19,6 @@ export enum NetworkId {
 
   FANTOM = 250,
   FANTOM_TESTNET = 4002,
-
-  Localhost = 1337,
 }
```

Comment:
I've removed `Localhost` from this enum. Thought it didn't really make sense here because localhost isn't a network, but a provider to connect to one of these networks. It's like putting `Alchemy` or `Infura` here. I think it was added for e2e tests? So going forward, when we spin up multiple local testnets for each network, we should populate the relevant `NODE_URL` env var for each network instead.

Reply:
Once the Dockerisation of local testnet is done (needs time...), it can be used for e2e tests, but it can also be used by developers, as they will have access to lots of ETH and OHM (through the faucet).

A local testnet will have the 1337 chain ID, and have its own set of contract addresses. Those contract addresses will change, so I have been working on a way of injecting those contract addresses (localhost only) dynamically at runtime. If you remove the `Localhost` enum, how will we know that the frontend is accessing a local testnet?

Reply:
Hmmm ok I see what you're saying now about the environment variables. That seems to work ok. Any thoughts on how to inject/override contract addresses? (Out of scope for this PR)

Reply:
Yeh I think overriding the contract address constants during the runtime of the tests would work alright. We'll look into it deeper when we get to it though :)